docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,182 @@
+
+Furthermore, the platform provides an `admin` application which manages authentication and authorization features.
+
+The `frontend` system is comprised of a demo application and the JavaScript integration of  the [Chat Plugin](sources/chat-plugin.md).
+
+Having that in mind, the docker containers or `Airy apps` which need to run in order for the Airy Core Platform to be fully functional, have the following naming convention:
+
+### Sources
+- sources-`SOURCE_NAME`-webhook - Receive messages from a specific source
+- sources-`SOURCE_NAME`-events-router - Process messages from a specific source
+- sources-`SOURCE_NAME`-sender - Send messages to a specific source
+
+### API
+- api-`ENTITY` - Backend services, exposed to the outside (admin, authentication, communication, etc.)
+
+### Webhook
+- webhook-`ENTITY` - Process events to be sent to external parties
+
+### Frontend
+- frontend-`ENTITY` - Frontend application, communicating with the `API` services
+
+
+## Requirements 
+
+The `Airy apps` should be started as separated docker containers and they must have access to several other underlying services, which need to be in place for the system to be fully functional:
+- `Kafka cluster`: Kafka, Zookeeper and the Schema registry. These three services comprise the kafka store and need to be in place as they are the base store for messages and events which are received and processed by the Airy Core Platform. Kafka needs Zookeeper to be in place to work and the Schema registry is an application which is the central schema repository for all the data written in kafka, used by the streaming apps inside the Airy Core Platform.
+- `PostgreSQL`: Holding a database which stores organization and user related data.
+- `Redis`: Data structure used as a simple message queue, for events which are sent to external webhooks.
+
+Apart from the underlying services, a docker runtime or a running Kubernetes cluster must also be available, for deploying the `Airy apps`.
+
+
+## Running on any infrastructure
+
+### Base infrastructure
+
+Depending on implementation of the user, different choices can be made regarding the implementation of the underlying services and the whole system in general. As first pre-requisites, the underlying services are needed to be running. They can be implemented as separate docker containers, in different Kubernetes clusters, on a separate virtual machines or physical servers. Here is a brief description on the implementation and how to fully achieve high availability.
+
+
+### The Kafka cluster
+
+The Kafka store can be comprised of multiple kafka brokers, zookeepers and Schema registry servers. The Airy apps can work with a single broker, however for running in production one can argue that this isn't enough. We recommend using at least five kafka brokers, having all the topics with a setting of replica factor 3. That way, even if two brokers become unavailable - the system will be fully operational. 
```

Comment:
```suggestion
The Kafka store requires multiple Kafka brokers, zookeepers, and Confluent Schema registry servers. We recommend using at least five Kafka brokers and  set the replication factor of _all_ topics to `3`.  So that even if two brokers would become unavailable at the same time, the system would still function. 
```